api/tests/app/delivery/test_send_to_providers.py

```diff
@@ -476,12 +476,12 @@ def test_get_html_email_renderer_handles_email_branding_without_logo(notify_api)
 
 @pytest.mark.parametrize('base_url, expected_url', [
     # don't change localhost to prevent errors when testing locally
-    ('http://localhost:6012', 'http://static-logos.notify.tools/filename.png'),
-    ('https://www.notifications.service.gov.uk', 'https://static-logos.notifications.service.gov.uk/filename.png'),
-    ('https://notify.works', 'https://static-logos.notify.works/filename.png'),
-    ('https://staging-notify.works', 'https://static-logos.staging-notify.works/filename.png'),
-    ('https://www.notify.works', 'https://static-logos.notify.works/filename.png'),
-    ('https://www.staging-notify.works', 'https://static-logos.staging-notify.works/filename.png'),
+    ('http://localhost:6012', 'http://localhost:6012/static-logo/filename.png'),
+    ('https://www.notifications.service.gov.uk', 'https://www.notifications.service.gov.uk/static-logo/filename.png'),
```

Comment:
Did we want to change this to a non gov.uk domain? Good for grep too so that we don't think we're still referring to them anywhere.